README.md

```diff
@@ -1 +1,85 @@
 # skygear-cloud-py-stripe
+
+This demo demostrates the use of Python Skygear Cloud Functions to make payment requests to Stripe.
+
+Skygear is a serverless platform that provides cloud functions hosting for Python and JavaScript routines. 
+
+Checkout the tutorial of this demo here: (Coming soon)
+
+## Run this project
+
+### Setting up
+
+1. Clone this project to your machine.
+
+      ```
+      git clone git@github.com:skygear-demo/skygear-cloud-py-stripe.git
```

Comment:
If you use https://github.com:skygear-demo/skygear-cloud-py-stripe.git, the user does not need a github account to clone.